debug/logs/2018-05-30 11-07-34.log

```diff
@@ -0,0 +1,5 @@
+42.0 APEX_CODE,FINEST;APEX_PROFILING,INFO;CALLOUT,INFO;DB,INFO;SYSTEM,DEBUG;VALIDATION,INFO;VISUALFORCE,INFO;WAVE,INFO;WORKFLOW,INFO
```

Comment:
Please remove the debug folder from the repository (add it to the .gitignore file and push your changes)